src/explore-education-statistics-admin/src/services/permissionService.ts

```diff
@@ -33,7 +33,7 @@ export type PreReleaseAccess = 'Before' | 'After' | 'Within' | 'NoneSet';
 export interface PreReleaseWindowStatus {
   access: PreReleaseAccess;
   start: Date;
-  end: Date;
+  scheduledPublishDate: Date;
```

Comment:
Again to keep consistent with current naming, would it be worth renaming this to `publishScheduled`?

Reply:
I did debate this a bit, but eventually chose to rename this for increased readability. 
To me, `publishScheduled` sounds like a boolean denoting whether a publish is scheduled or not, and `ScheduledPublishDate` is a bit more descriptive. 

I appreciate it sort of implicitly *is* actually a boolean because we hinge logic off of whether it's null or not, but I had to read a fair bit of code to check that assumption. 

Certainly happy to revert the name though in the cause of consistency if you prefer?